src/App.tsx

```diff
@@ -48,10 +48,6 @@ export const App = (): React.ReactElement => {
             />
             <meta property="og:image:width" content="400" />
             <meta property="og:image:height" content="400" />
-            <meta
-              property="og:url"
```

Comment:
Why not just add the full path to the page.  Looks like you're just adding the domain here instead of the full path to this page. Which explains the behavior you were getting. Wanna try re adding it with the full path?

Reply:
If we don't include the query params, then facebook will break sharing scenarios by link. If we do include the query params as the og:url, then we're defeating the purpose of og:url (which is to enforce sharing clean urls)

Reply:
Hm, I guess just plopping in window.location.href would unify www vs no www and include the method, which is somewhat useful

Reply:
Done